Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Block_IP_-_Custom_Block_Rule_README.md

```diff
@@ -0,0 +1,44 @@
+This playbook blocks IP addresses using Custom Block Rules in Checkpoint Firewall.
+The playbook receives malicious IP addresses as inputs, creates a custom bi-directional rule to block them, and publish the configuration.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Checkpoint - Publish&Install configuration
+
+### Integrations
+* CheckPointFirewallV2
+
+### Scripts
+* Print
+
+### Commands
+* checkpoint-logout
+* checkpoint-host-add
+* checkpoint-access-rule-list
+* checkpoint-access-rule-add
+* checkpoint-login-and-get-session-id
+* checkpoint-show-objects
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| IP | Array of malicious IPs to block. |  | Required |
```

Comment:
Just array or also command delimited? Please provide an example

Reply:
The context array - I'll try to add an example and check if the ["1.1.1.1","2.2.2.2"] option as an input will work (instead of providing context path like ${IP})

Reply:
added support also for comma separated values.